examples/linear_model/plot_orthogonal_regression_nonanalytic_behavior.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
Can you make an open issue to improve this notebook after merging?

Reply:
added #43